WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift

```diff
@@ -263,4 +270,21 @@ private extension NoResultsViewController {
         return String(format: buttonIdFormat, string)
     }
 
+    // MARK: - `WPAnimatedBox` resource management
+
+    private func startAnimatingIfNeeded() {
+        guard let animatedBox = accessorySubview as? WPAnimatedBox else { return }
+        animatedBox.animate(afterDelay: 0.1)
+    }
+
+    private func stopAnimatingIfNeeded() {
+        guard
+            let accessorySubview = accessorySubview,
+            let animatedBox = accessorySubview as? WPAnimatedBox
```

Comment:
Can these two `let`s be consolidated into one line, like above in `startAnimatedIfNeeded`?

`guard let animatedBox = accessorySubview as? WPAnimatedBox`

Reply:
Good call! It also addresses some of the formatting feedback, albeit indirectly. I'm happy to make this change.